drivers/starburst/resources/metabase-plugin.yaml

```diff
@@ -20,7 +20,7 @@ driver:
         - name: catalog
           placeholder: tpch
           display-name: Catalog
-          required: false
+          required: true
```

Comment:
What if we want to execute a federated query across catalogs or get the data from two different catalogs? Is it needed then to create a separate connection for each catalog?

Reply:
I remember there was a discussion during the call with Metabase about it, but completely forgot what was the outcome. They mentioned something about a workaround.